i18n/locales/source/blockly-mooc/common.json

```diff
@@ -750,7 +750,7 @@
   "emptyFunctionBlocksErrorMsg": "The function block needs to have other blocks inside it to work.",
   "emptyFunctionalBlock": "You have a block with an unfilled input.",
   "emptySection": "Your section doesn't have any students! Go to the [Manage Students]({url}) tab to add some.",
-  "emptySurveyOverviewTable": "Because this survey is anonymous, we can only show aggregated results once there are at least 5 submissions.",
```

Comment:
I'm curious what's the difference between this common.json file and the en_us.json file? Is this one generated?

Reply:
Great question... I am unclear, to be honest. I believe putting it in en_us starts the translation pipeline, but just updated it both places to be safe. I can look into this!

Reply:
Thanks! I think I've only been adding to `en_us.json`, so it'd be great to know if I need to add to both in the future

Reply:
you're fine to only add it to `en_us.json`; the i18n build takes care of compiling it to everywhere it needs to go.

Reply:
Note in particular that the history of the file has it being updated by an automated process https://github.com/code-dot-org/code-dot-org/commits/d42e1b8d01889e1f7b99ce8d682089e4f421c55f/i18n/locales/source/blockly-mooc/common.json